### DISCUSSION TITLE
Application Approved but cant access benefits

### DISCUSSION THREAD

**Discussion by ImanBanerjee** [community/community#76218] [GitHub Education]
### Hi everyone,

Hey my applications is showing approved but i cant access any benefits can someone help?

**Comment by ghost** [comment]
Hello @ImanBanerjee 

I suggest to wait few days first and let github process request. 

if you can't see benefits after few days of waiting you should contact github support team to resolve this issue. few more person asked same question before you and no ona had answer on that. so just wait few more days and then contact github support. Also good to take a look on this [answer](https://github.com/orgs/community/discussions/76041#discussioncomment-7612753), it can help

hope this helps.
